client/src/components/pages/personalProfile/index.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react'
+import { Form, Input } from 'antd'
+import Button from '../../utils/Button'
+
+import 'antd/dist/antd.css'
```

Comment:
@AbdallahAmmar96 It might be better to put this external module's css import into App.css - then you only have to do this import once, instead of doing it in every file that uses antd components.